app/ui-react/syndesis/src/modules/integrations/components/editor/EditorStepsWithToolbar.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  ConnectionsListView,
+  IActiveFilter,
+  IConnectionsListViewProps,
+  IFilterType,
+  ISortType,
+} from '@syndesis/ui';
+import { WithListViewToolbarHelpers } from '@syndesis/utils';
+import * as React from 'react';
+import { Translation } from 'react-i18next';
+import i18n from '../../../../i18n';
+import resolvers from '../../../connections/resolvers';
+import { EditorSteps, IEditorStepsProps } from './EditorSteps';
+import { IUIStep } from './interfaces';
+
+function getFilteredAndSortedEditorSteps(
+  steps: IUIStep[],
+  activeFilters: IActiveFilter[],
+  currentSortType: ISortType,
+  isSortAscending: boolean
+) {
+  let filteredAndSortedEditorSteps = steps;
+  activeFilters.forEach((filter: IActiveFilter) => {
+    const valueToLower = filter.value.toLowerCase();
+    filteredAndSortedEditorSteps = filteredAndSortedEditorSteps.filter(
+      (e: IUIStep) => e.name.toLowerCase().includes(valueToLower)
+    );
+  });
+
+  filteredAndSortedEditorSteps = filteredAndSortedEditorSteps.sort(
+    (miA, miB) => {
+      const left = isSortAscending ? miA : miB;
+      const right = isSortAscending ? miB : miA;
+      return left.name.localeCompare(right.name);
+    }
+  );
+
+  return filteredAndSortedEditorSteps;
+}
+
+const filterByName = {
+  filterType: 'text',
+  id: 'name',
+  placeholder: i18n.t('shared:filterByNamePlaceholder'),
+  title: i18n.t('shared:Name'),
+} as IFilterType;
+
+const filterTypes = [filterByName];
+
+const sortByName = {
+  id: 'name',
+  isNumeric: false,
+  title: i18n.t('shared:Name'),
+} as ISortType;
+
+const sortTypes: ISortType[] = [sortByName];
+
+export interface IEditorStepsWithToolbarProps
+  extends IEditorStepsProps,
+    Pick<IConnectionsListViewProps, 'createConnectionButtonStyle'> {
+  children?: any;
+}
+
+export class EditorStepsWithToolbar extends React.Component<
+  IEditorStepsWithToolbarProps
+> {
+  public static defaultProps = {
+    includeHidden: false,
```

Comment:
I think we want to show the hidden things in this particular case, no?  AFAIK this controls if the API provider connection is displayed and Timer and WebHook.  Actually, since this is now a component dedicated to just this page, I don't think we even need the flag.

Reply:
Yeah, the page works as I'd expect, I think this might be a leftover.

Reply:
Yeah this is just left over. Wasn't really looking at it from this perspective tbh, but it makes sense, I'll remove it.